### PR TITLE
Added debug verbosity

### PR DESCRIPTION
**Pull request by bontreger**
5th point in this blog from Tim Appnel https://www.ansible.com/blog/ansible-best-practices-essentials

### CODE REVIEW

File: playbooks/README.adoc [hunk 1]
```diff
@@ -127,3 +127,24 @@ The same approach is also possible with `include_role` but requires additionally
...
+
+Rationale:: Debug messages are useful during testing and development, and can be useful to retain as playbooks go into production, but can cause concerns as they are shared between teams.
+
+Examples::
++
+.Adding verbosity to debug messages
+[source, yaml]
+----
+- debug:
+   msg: "Don't make messages always display"
```
Comment (ericzolf) [inline playbooks/README.adoc:144]:
```suggestion
- name: don't make messages always display
  debug:
    msg: "This message will clutter your log in production"
```
The indentation was slightly off and we recommend to always have a name

File: playbooks/README.adoc [hunk 2]
```diff
@@ -127,3 +127,24 @@ The same approach is also possible with `include_role` but requires additionally
 ----
 
 ====
+
+== Use the verbosity parameter with debug statements
+[%collapsible]
+====
+Explanations:: Debug messages should have a verbosity defined as appropriate for the message.
+
+Rationale:: Debug messages are useful during testing and development, and can be useful to retain as playbooks go into production, but can cause concerns as they are shared between teams.
```
Comment (ericzolf) [inline playbooks/README.adoc:136]: I would rather say that it clutters your log with possibly a lot of needless information which might confuse users.

File: playbooks/README.adoc [hunk 3]
```diff
@@ -127,3 +127,24 @@ The same approach is also possible with `include_role` but requires additionally
...
++
+.Adding verbosity to debug messages
+[source, yaml]
+----
+- debug:
+   msg: "Don't make messages always display"
+
+- debug:
+   msg: "Show this only when verbosity is 2 or more"
+   verbosity: 2
```
Comment (ericzolf) [inline playbooks/README.adoc:148]:
```suggestion
- name: this message will only appear when verbosity is 2 or more
  debug:
    msg: "Some more debug information if needed"
    verbosity: 2
```

**Review by ericzolf — CHANGES_REQUESTED** [review]
Looks mostly good to me and valuable. Few trifles to fix though. Thanks!

**Comment by ericzolf** [comment]
I think, my one comment hasn't been addressed/answered.

File: playbooks/README.adoc [hunk 4]
```diff
@@ -127,3 +127,26 @@ The same approach is also possible with `include_role` but requires additionally
 ----
 
 ====
+
+== Use the verbosity parameter with debug statements
+[%collapsible]
+====
+Explanations:: Debug messages should have a verbosity defined as appropriate for the message.
+
+Rationale:: Debug messages are useful during testing and development, and can be useful to retain as playbooks go into production for future troubleshooting. However, log messages will clutter your output, which can confuse users with non-relevant information.
```
Comment (mophahr) [inline playbooks/README.adoc:136]:
```suggestion
Rationale::
Debug messages are useful during testing and development, and can be useful to retain as playbooks go into production for future troubleshooting.
However, log messages will clutter your output, which can confuse users with non-relevant information.
```

**Review by ericzolf — APPROVED** [review]

**Comment by tima** [comment]
Love it. Please include.

**Review by sean-m-sullivan — APPROVED** [review]
I like it.

**Comment by mophahr** [comment]
Approved in the CoP meeting.

**Pull request merged**